code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -398,11 +398,19 @@
 	if(slot == ITEM_SLOT_FEET)
 		for(var/mob/living/occupant in occupants)
 			occupant.forceMove(user.drop_location())
-			user.visible_message("<span class='warning'>[user] recoils as something slithers out of [src].</span>", "<span class='userdanger'>You feel a sudden stabbing pain in your [pick("foot", "toe", "ankle")]!</span>")
-			user.Knockdown(20) //Is one second paralyze better here? I feel you would fall on your ass in some fashion.
-			user.apply_damage(5, BRUTE, pick(BODY_ZONE_R_LEG, BODY_ZONE_L_LEG))
-			if(istype(occupant, /mob/living/simple_animal/hostile/retaliate/poison))
-				user.reagents.add_reagent(/datum/reagent/toxin, 7)
+			var/friendly_friends = FALSE
+			if(istype(occupant, /mob/living/simple_animal/hostile))
```

Comment:
Could you substitute this `istype` check with a generic variable or trait describing this kind of behavior you can apply to different things?

Reply:
is a "snake_boot_snowflake_check" variable really necessary?

Reply:
Definitely not, but that's because that's not a generic variable.
Now a `is_hostile_mob` is.